### ISSUE TITLE
Language detected along episode title

### ISSUE THREAD

**Issue by duramato** [guessit-io/guessit#574]
For this german Scene release, the episode title gets detected as `Treibjagd German` when it's only, `Treibjagd`. `German is the main audio language`, and DL indicates that has multiple audio tracks, as correctly detected `Multiple languages`.
```
For: S.W.A.T.2017.S01E21.Treibjagd.German.Dubbed.DL.AmazonHD.x264-TVS
GuessIt found: {
    "title": "S W.A.T.",
    "year": 2017,
    "season": 1,
    "episode": 21,
    "episode_title": "Treibjagd German",
    "language": "Multiple languages",
    "streaming_service": "Amazon Prime",
    "other": "HD",
    "video_codec": "H.264",
    "release_group": "TVS",
    "type": "episode"
}
```

**Comment by Toilal** [comment]
It should guess `subtitle_language: German` to fix the episode title.

**Comment by duramato** [comment]
>It should guess subtitle_language: German to fix the episode title.
But `German` is not a subtitle track? It's one of the audio languages (`German.Dubbed`), the other audio language isn't explicit from the release name (`DL` just indicated Dual Audio).

**Comment by Toilal** [comment]
Right, it should be `language : [German, Multi]`

**Comment by ratoaq2** [comment]
Fixed by 44f7d60e0894ef0cb0f1654e2b320365f1076468
